packages/mgt/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -195,6 +197,31 @@ export class MgtAgenda extends MgtTemplatedComponent {
     super.disconnectedCallback();
   }
 
+  /**
+   * returns dir attribute on body
+   *
+   * @private
+   * @memberof MgtAgenda
+   */
+  private updateDirection() {
+    let direction = LocalizationHelper.getDirection();
+    if (direction == 'rtl') {
+      this.classList.add('rtl');
+    }
+  }
+
+  /**
+   * Matches string to user provided one and returns
+   *
+   * @protected
+   * @param {*} stringKey
+   * @returns
+   * @memberof MgtPeoplePicker
+   */
+  protected getString(stringKey) {
```

Comment:
The `getString` implementation should go in `BaseComponent` imo. Reduces the need to redefine the function in every component.